packages/react/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -65,13 +65,16 @@ const CheckboxGroup: React.FC<CheckboxGroupProps> = ({
   ) : null;
 
   const fieldsetClasses = cx(`${prefix}--checkbox-group`, className, {
-    [`${prefix}--checkbox-group--${orientation}`]: orientation === 'horizontal',
     [`${prefix}--checkbox-group--readonly`]: readOnly,
     [`${prefix}--checkbox-group--invalid`]: !readOnly && invalid,
     [`${prefix}--checkbox-group--warning`]: showWarning,
     [`${prefix}--checkbox-group--slug`]: slug,
   });
 
+  const orientationClass =
+    orientation === 'horizontal'
+      ? `${prefix}--checkbox-group--${orientation}`
+      : '';
```

Comment:
```suggestion
      : undefined;
```
LGTM! Let's just change to `undefined` to hide the empty class.

![Screenshot 2024-11-18 at 10 38 00](https://github.com/user-attachments/assets/89c28c08-5452-47fe-85a3-2784dc4629df)